### ISSUE TITLE
Devendor all example-clients and tools

### ISSUE THREAD

**Issue by dvzrv** [jackaudio/jack2#805]
## Describe the bug

The [jack-example-tools](https://github.com/jackaudio/jack-example-tools) will soon replace the use and need of [example-clients](https://github.com/jackaudio/example-clients) and [tools](https://github.com/jackaudio/tools) in the context of the jack1 project.
For the new project to be able to also fully replace the [example-clients](https://github.com/jackaudio/jack2/tree/0727bbc386288f6ff5cb6876d2321836d562221e/example-clients) and [tools](https://github.com/jackaudio/jack2/tree/0727bbc386288f6ff5cb6876d2321836d562221e/tools) of the jack2 project, they need to be consolidated with the new code base and afterwards fully devendored from this project.

This way we will be able to provide them for any implementation of jack (also pipewire, see relevant upstream ticket: https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/1666) in the future and at the same time enable for an easier development of the jack2 code.

## Environment

* **JACK Version:** 1.9.19
* **Operating System:** any
* **Installation:** n/a

## Steps To Reproduce

* compare jack1 and jack2 installations: the example-clients and tools differ (slightly)
* install pipewire-jack: the example-clients and tools are missing and can not be installed, because when pipewire is built as a full jack replacement, it conflicts with jack1 and jack2 (depending on package splitting)

### Expected vs. actual behavior

The jack-example-tools can be installed while switching jack implementations (jack1, jack2 or pipewire-jack).
Currently only different versions of the example-clients and tools can be installed alongside jack1 and jack2, but are missing when using pipewire-jack.

**Comment by h1z1** [comment]
TLDR on "devendored" ??   First search result is this bug report, the others are just as baffling.

If I wanted pipewire I'd install pipewire.
